### DISCUSSION TITLE
[Question][Hw1] When there is no page linking?

### DISCUSSION THREAD

**Discussion by yongwoo36** [prosyslab-classroom/cs524-program-analysis#405] [Q&A]
What is the page rank of a page that has no incoming linking?
It should be 0 according to equation in doc, or it maintain the initial value 1?

**Comment by hyunsukimsokcho** [comment]
> What is the page rank of a page that has no incoming linking?

In each iteration, a pagerank of a page `p` in a web is computed by a sum of two factors:
1. a set of pages linking to `p` (see the equation)
2. a set of pages that has no outgoing link (see the third bullet point)

Rather than giving the answer to the exact case, we would like to restate the problem as above. Please read the docs carefully and hope this helps.


> It should be 0 according to equation in doc, or it maintain the initial value 1?

It really depends. It can be 0, 1, or neither of the two.

**Comment by yongwoo36** [comment]
Oh, I miss the third bullet point. Thanks!
